origin-js/test/ipfs_store.test.js

```diff
@@ -65,6 +66,16 @@ describe('Listing IpfsDataStore load', () => {
     expect(listing.ipfs.data).to.deep.equal(validListing)
   })
 
+  it(`Should load a listing with no dappSchemaId`, async () => {
+    const listingNoDappSChemaId = Object.assign({}, validListing)
+    delete listingNoDappSChemaId.dappSchemaId
+    mockIpfsService.loadObjFromFile = sinon.stub().resolves(listingNoDappSChemaId)
+    mockIpfsService.rewriteUrl = sinon.stub().returns('http://test-gateway')
+
+    const listing = await store.load(LISTING_DATA_TYPE, 'TestHash')
+    expect(listing.dappSchemaId).to.equal(undefined)
+  })
+
```

Comment:
Would be great to see `dappSchemaId` added to a test listing data set and make sure it gets returned by a call to `getListing()` in the marketplace API

Reply:
Good point. Let me add a unit test to marketplace.